src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -223,36 +244,82 @@ private function fixFunction(Tokens $tokens, int $startFunctionIndex): bool
             }
         }
 
+        $argumentCount = 1;
+        $nestedMultiline = false;
+
         for ($index = $endFunctionIndex - 1; $index > $startFunctionIndex; --$index) {
             $token = $tokens[$index];
+            $oldIndex = $index;
 
             if ($token->equals(')')) {
                 $index = $tokens->findBlockStart(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
 
+                if ('ignore' !== $this->configuration['on_nested_multiline'] && $this->doesBlockContainNewLine($tokens, $oldIndex, $index)) {
```

Comment:
Maybe we can avoid looking for nested newlines if we already found one:
```suggestion
                if (
                    !$nestedMultiline
                    && 'ignore' !== $this->configuration['on_nested_multiline']
                    && $this->doesBlockContainNewLine($tokens, $oldIndex, $index)
                ) {
```